Admin/FragmentAdmin.php

```diff
@@ -0,0 +1,273 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ArticleBundle\Admin;
+
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\AdminBundle\Route\RouteCollection;
+use Sonata\AdminBundle\Show\ShowMapper;
+use Sonata\ArticleBundle\FragmentService\FragmentServiceInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author  Hugo Briand <briand@ekino.com>
+ */
+final class FragmentAdmin extends AbstractAdmin
+{
+    /**
+     * @var FragmentServiceInterface[]
+     */
+    private $fragmentServices = array();
```

Comment:
A FragmentService is like a BlockService, it provide a class to configure them, and render them.